README.md

```diff
@@ -49,6 +49,27 @@ Once you clone Hemi from Github, you need to sync its submodules in order to use
 
 Hemi is a header-only library, so there is no further installation required. Simply include the hemi headers for the features you need. See the examples for demonstration.
 
+### Optional CMake Installation
+
+If you would like to use cmake to manage the installation
+of this code, you can install with:
+
+    > mkdir build && cd build
```

Comment:
I suspect this would work without local installation using CPM. Are you familiar with CPM (`CPMAddPackage()`)? If so, perhaps test out whether this works and provide an example.